cheeseshop/dbapi.py

```diff
@@ -311,10 +311,79 @@ class CsGoSteamId(object):
         await conn.execute('''
             CREATE TABLE cs_go_steam_ids(
                 id serial PRIMARY KEY,
-                steam_id text UNIQUE NOT NULL
+                steam_id text UNIQUE NOT NULL,
+                print_name text,
```

Comment:
I dont think we can have this column here as names are changeable. The two options are make this a list (although theres some issues there perf wise) or break this out in to its own table we refer in to (probably best).

Reply:
Will do.